solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/impl/FieldLengthFeature.java

```diff
@@ -121,11 +121,12 @@ public FieldLengthFeatureScorer(FeatureWeight weight,
 
       @Override
       public float score() throws IOException {
+        if (norms == null) {
```

Comment:
> Does it make sense to just construct and return a constant 0 value feature scorer if this is true, instead of checking for all score() calls?

I don't see any way to instantiate a `ValueFeatureScorer` here, we could have a similar class inside `FieldLengthFeatureWeight`, but I don't think will be better wrt check if `norms == null`, what do you think? 

> Do we have a similar null exception for the fieldvalue feature?
I think it's different here, in FieldValue you want to check if the current field has a field, why here we are checking if the segment has the norms..